scripts/site.sh

```diff
@@ -55,7 +62,7 @@ EOF
 }
 
 function _docker() {
-  docker-compose -f "$rootDir/docker-compose.yml" "$@"
+  $composeCmd -f "$rootDir/docker-compose.yml" "$@"
```

Comment:
You can just hardcode this to `docker compose` - I just haven't gotten around to updating this